banzai/utils/date_utils.py

```diff
@@ -125,13 +125,14 @@ def get_dayobs(timezone):
 
 def validate_date(s):
     try:
-        return datetime.datetime.strptime(s, TIMESTAMP_FORMAT)
```

Comment:
Merge issue, this should be changed back

Reply:
Fixed.